elasticsearch/values.yaml

```diff
@@ -184,3 +192,11 @@ fullnameOverride: ""
 
 # https://github.com/elastic/helm-charts/issues/63
 masterTerminationFix: false
+
+lifecycle: {}
+  # preStop:
+  #   exec:
+  #     command: ["/bin/bash","/preStop"]
+  # preStop:
```

Comment:
This example is the same as above. I think this one was meant to be `postStart`.

Reply:
done https://github.com/elastic/helm-charts/pull/197/commits/db535e2ae6c6e4d7642ab395b79f65779ed8935f, I've also moved `hooks` value closer to the `lifecycle` to make it more clear where should be defined content of scripts